.gitignore

```diff
@@ -4,13 +4,15 @@
 /node_modules
 /.pnp
 .pnp.js
+bun.lockb
```

Comment:
This should always be included in commits. It is the lock file, can you remove it from git ignore?